src/preprocessing/passes/ackermann.cpp

```diff
@@ -187,9 +188,124 @@ void collectFunctionsAndLemmas(FunctionToArgsMap& fun_to_args,
 
 /* -------------------------------------------------------------------------- */
 
+bool needsReplace(TNode term)
+{
+  if (term.getType().isSort() && !(term.getKind() == kind::BOUND_VARIABLE))
+    return true;
+  return false;
+}
+
+/* Given the lowest capacity requirements for each uninterpreted sorts, assign
+ * a sufficient bit-vector size. Get the converting map */
+void collectUSortsToBV(USortToBVSizeMap& usortCardinality,
+                       vector<TNode>& vec,
```

Comment:
```suggestion
                       const vector<TNode>& vec,
```

We are not modifying `vec` (as far as I can tell), so this should be a `const` reference. Also, can we choose a more descriptive variable name? If that's the case, I would also move `vec` to be the first argument (to have a separation between arguments that are just inputs and arguments that are modified), although that's subjective.